xwiki-commons-core/xwiki-commons-xml/src/main/resources/META-INF/components.txt

```diff
@@ -7,5 +7,5 @@ org.xwiki.xml.internal.html.filter.ControlCharactersFilter
 org.xwiki.xml.internal.html.filter.AttributeFilter
 org.xwiki.xml.internal.html.filter.UniqueIdFilter
 org.xwiki.xml.internal.html.DefaultHTMLCleaner
-org.xwiki.xml.internal.LocalEntityResolver
```

Comment:
Since you removed it, it cannot be used anymore and thus I guess it means it's not used anymore in xwiki commons/rendering/platform. Thus we should be able to remove the `LocalEntityResolver` class altogether (or move it to a legacy module - https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HBackwardCompatibility).

@carlosame I could take over from there and do the work for the legacy move.

Reply:
> it cannot be used anymore

Well it could be used again if the user is told to edit the `components.txt` file. See my other comment.